### PR TITLE
fixed link and added consistency

### LINKED ISSUE #611: should open to new tab
clicking on vega view open the link on the same tab, i think it should open the link in another tab so that one should not have to leave vega app 
Screenshot from 2020-02-18 10-08-10

There are other links also like hover link in this image
Screenshot from 2020-02-18 10-11-47

### PR DESCRIPTION
**Pull request by ashu8912**
fixes #611

### CODE REVIEW

File: src/components/header/help-modal/index.tsx [hunk 1]
```diff
@@ -67,8 +67,12 @@ const HelpModal = () => {
       </a>
       <h2>Tip</h2>
       <p>
-        You can access Vega, Vega-Lite, and the <a href="https://vega.github.io/vega/docs/api/view/"> Vega view</a> at{' '}
-        <code>(VEGA_DEBUG)</code> using your browser's developer console.
+        You can access Vega, Vega-Lite, and the{' '}
+        <a href="https://vega.github.io/vega/docs/api/view/" target="_blank" rel="noopener noreferrer">
+          {' '}
```
Comment (siddhant1) [inline src/components/header/help-modal/index.tsx]: Can you remove this?

Comment (siddhant1) [inline src/components/header/help-modal/index.tsx]: Can you please remove this?

File: src/components/header/help-modal/index.tsx [hunk 2]
```diff
@@ -67,8 +67,12 @@ const HelpModal = () => {
       </a>
       <h2>Tip</h2>
       <p>
-        You can access Vega, Vega-Lite, and the <a href="https://vega.github.io/vega/docs/api/view/"> Vega view</a> at{' '}
-        <code>(VEGA_DEBUG)</code> using your browser's developer console.
+        You can access Vega, Vega-Lite, and the{' '}
```
Comment (domoritz) [inline src/components/header/help-modal/index.tsx]: Do we need space before and after the opening tag?

Comment (siddhant1) [reply src/components/header/help-modal/index.tsx]: Yeah , that is for some space between the <code> and the 'the'.

Comment (siddhant1) [reply src/components/header/help-modal/index.tsx]: I am opening a new issue to improve the css of the help modal

File: src/components/header/help-modal/index.tsx (hunk 1 above)
Comment (siddhant1) [inline src/components/header/help-modal/index.tsx]: Remove this

Comment (ashu8912) [reply src/components/header/help-modal/index.tsx]: thanks @siddhant1 for mentioning this

File: src/components/header/help-modal/index.tsx (hunk 2 above)
Comment (ashu8912) [reply src/components/header/help-modal/index.tsx]: yes

**Comment by ashu8912** [comment]
@domoritz this can be merged now

**Comment by domoritz** [comment]
Thank you both

**Pull request merged**